yarn.lock

```diff
@@ -3383,22 +3383,22 @@
   resolved "https://registry.yarnpkg.com/@react-native-segmented-control/segmented-control/-/segmented-control-2.4.1.tgz#ea612c8b12bfea7712c7ea456b7b98dfc6748485"
   integrity sha512-c8Q7TP14CD6vJ38VO5SWoIcxfwLPIz/4zRKJnJOJ0xKkMxkrUu089m86Y0dSvo+T3c1NQyvBFRtKWrPqbqG6tg==
 
-"@react-native/assets-registry@0.74.75", "@react-native/assets-registry@~0.74.75":
-  version "0.74.75"
-  resolved "https://registry.yarnpkg.com/@react-native/assets-registry/-/assets-registry-0.74.75.tgz#58779e2a2dd0f012ad4286ef4cd5f59c53cf7cf6"
-  integrity sha512-1TQNmRmX0F0vhYrjqyJstmPtewFEgy9HOuE8cssuqltWbHlAqqTt2hXVYMZW5wyUL7fRdKcM0FvZJKGsT1yf+A==
+"@react-native/assets-registry@0.74.76", "@react-native/assets-registry@~0.74.75":
```

Comment:
nit: some expo packages have dependencies to these @react-native dependencies and i used to update them all to prevent any resolution accidents.
but maybe we can do that after 0.74 final releases.

Reply:
Updated 👍